config/core/test-configs.yaml

```diff
@@ -2158,6 +2164,7 @@ test_configs:
   - device_type: r8a7796-m3ulcb
     test_plans: &r8a7796-m3ulcb_test-plans
       - baseline
+      - kselftest-lib
```

Comment:
Please drop this line, it's conflicting with #672.

Reply:
done.